src/app/components/bitcoin-fees-list/bitcoin-fees.utils.ts

```diff
@@ -0,0 +1,144 @@
+import { type BtcCryptoAssetBalance, type MarketData, Money } from '@leather.io/models';
+import { type UtxoResponseItem } from '@leather.io/query';
+import {
+  baseCurrencyAmountInQuote,
+  capitalize,
+  createMoney,
+  formatMoneyPadded,
+  i18nFormatCurrency,
+} from '@leather.io/utils';
+
+import type { TransferRecipient } from '@shared/models/form.model';
+
+import type { FeeDisplayInfo, FeeType } from '@app/common/fees/use-fees';
+import {
+  type DetermineUtxosForSpendArgs,
+  determineUtxosForSpend,
+  determineUtxosForSpendAll,
+} from '@app/common/transactions/bitcoin/coinselect/local-coin-selection';
+import { getSizeInfo } from '@app/common/transactions/bitcoin/utils';
+
+export function getBitcoinFee(determineUtxosForFeeArgs: DetermineUtxosForSpendArgs) {
+  try {
+    const { fee } = determineUtxosForSpend(determineUtxosForFeeArgs);
+    return fee;
+  } catch (error) {
+    return null;
+  }
+}
+
+export function getBitcoinSendMaxFee(determineUtxosForFeeArgs: DetermineUtxosForSpendArgs) {
+  try {
+    const { fee } = determineUtxosForSpendAll(determineUtxosForFeeArgs);
+    return fee;
+  } catch (error) {
+    return null;
+  }
+}
+
+export function getApproximateFee({
+  feeRate,
+  recipients,
+  utxos,
+}: {
+  feeRate: number;
+  recipients: TransferRecipient[];
+  utxos: UtxoResponseItem[];
+}) {
+  const size = getSizeInfo({
+    inputLength: utxos.length + 1,
+    recipients,
+  });
+  return Math.ceil(size.txVBytes * feeRate);
+}
+
+export function formatBitcoinFeeForDisplay({
+  rawFee,
+  marketData,
+}: {
+  rawFee: RawFee;
+  marketData: MarketData;
+}): FeeDisplayInfo {
+  function getFiatFeeValue(fee: number) {
+    return i18nFormatCurrency(
+      baseCurrencyAmountInQuote(createMoney(Math.ceil(fee), 'BTC'), marketData)
+    );
+  }
+
+  const { type, baseUnitsFeeValue, feeRate, time } = rawFee;
+
+  return {
+    feeType: type,
+    feeRate: feeRate ?? 0,
+    baseUnitsValue: baseUnitsFeeValue ?? 0,
+    titleLeft: capitalize(type),
```

Comment:
Building on earlier comment, why don't we do the `capitalize` in the view too? How we format/present data is a concern of the UI so better lives where it's set to a component.

My thinking is that if I as a dev need to go and make visual changes, they can be mostly scoped to a set of `.layout` components. But in this file, we have a mix of mathematics and formatting, that I can't help but feel should live separately.

e.g. `<style.span>{capitalize(type)}</styled.span>`

Then, the utils file knows nothing about where, or how, the data needs to be displayed, focusing on accurate data only.

What's your thinking for putting it here, and are there reasons I'm missing why it's beneficial to group it here?